### PR TITLE
fix(workflows): Adds prepare release commit push to bugfix branch

### PR DESCRIPTION
**Pull request by rafaelmag110**
## WHAT

Adds prepare release commit push to bugfix branch

## WHY
It isn't pushing to branch and the commit is lost

### CODE REVIEW

**Comment by rafaelmag110** [comment]
Requesting review
@wolf4ood

**Review by wolf4ood — APPROVED** [review]

File: .github/workflows/publish-bugfix-release.yml [hunk 1]
```diff
@@ -62,6 +62,8 @@ jobs:
       contents: write
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```
Comment (ndr-brt) [inline .github/workflows/publish-bugfix-release.yml:66]: this shouldn't be necessary unless there's the need to checkout to an existent commit (like it's done in the `helm-release` job)

File: .github/workflows/publish-bugfix-release.yml [hunk 2]
```diff
@@ -91,6 +93,8 @@ jobs:
         run: |
           git add gradle.properties $(find charts -name Chart.yaml) $(find charts -name README.md)
           git commit --message "Prepare release ${{ inputs.version }}"
+          
+          git push origin ${{ github.ref_name }}
```
Comment (ndr-brt) [inline .github/workflows/publish-bugfix-release.yml:97]:
```suggestion
          git push
```

Comment (rafaelmag110) [reply .github/workflows/publish-bugfix-release.yml:97]: Wasn't sure just the push would work fine so decided to pass the `ref_name`. Then I wasn't sure the branch would be discoverable and then added the `fetch-depth: 0`.

Nevertheless I tested this in a private repo and the simpler version works, so i'll change.

**Comment by rafaelmag110** [comment]
@ndr-brt Please merge if the new commit its ok.

**Pull request merged**